ReadMe.md

```diff
@@ -1 +1,3 @@
-Financial Calcs
\ No newline at end of file
+Financial Calcs
```

Comment:
Can u change this to financial calculations

Reply:
completed suggested changes @muthinacharmila, please review and merge the changes.